.api/v1.5.0/schema.pbiviz.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
right - I tried that at the time. If I recall correctly it still used the global one. But I'll take another look.